plugins/config/lib/configService.js

```diff
@@ -2264,6 +2358,18 @@ function getPluginConfiguration(identifier, pluginLocation, serverSettings,produ
 };
 exports.getPluginConfiguration = getPluginConfiguration;
 
+function getAllowedPlugins(options, username, identifier, pluginLocation) {
+  let filename = "allowedPlugins.json"
+  let relativePath = identifier + "/plugins"
+  let serverSettings = options.serverConfig
+  let productCode = options.pluginLoader.options.productCode
+  let directories = makeConfigurationDirectoriesStructInner(serverSettings, productCode, username, pluginLocation);
+  directories._pluginID = encodeDirectoryName(identifier)
+  return getMergeJson(relativePath, filename, directories)
```

Comment:
Scope is required but omitted

Reply:
https://github.com/zowe/zlux-server-framework/pull/153#discussion_r332650846